src/liballoc/rc.rs

```diff
@@ -10,7 +10,8 @@
 
 #![allow(deprecated)]
 
-//! Single-threaded reference-counting pointers.
+//! Single-threaded reference-counting pointers. "Rc" stands for "Reference
```

Comment:
I think of Rc as "reference counted", not "reference counter" personally, and Arc as "atomically reference counted".